prov/rxm/src/rxm_rma.c

```diff
@@ -37,7 +37,7 @@ typedef ssize_t rxm_rma_msg_fn(struct fid_ep *ep_fid,
 
 static inline ssize_t
 rxm_ep_rma_reg_iov(struct rxm_ep *rxm_ep, const struct iovec *msg_iov,
-		   void **desc, void **mr_storage,
+		   void ***desc, void **mr_storage,
```

Comment:
I'm not sure mere mortals can process a pointer to a pointer to a pointer, especially when the pointer to the pointer to the pointer is set to point to the pointer to the pointer that's also passed in as a parameter.  (See what I mean? :)  We need _something_ a little simpler here.